pyproject.toml

```diff
@@ -55,6 +55,7 @@ mypy = "^1.6.0"
 pycodestyle = "^2.11.1"
 types-pyyaml = "^6.0.12.12"
 types-requests = "^2.31.0.9"
+pyfakefs = "^5.3.0"
```

Comment:
What is this needed for?

Reply:
creating a fake file system that mimics maxwell. It's then used to test initialize paths